Future + Feature (without screenshot)

huongvu2312 opened the pull request:
Trying to fix the git flow took me too much time, remove the project and clone it again didn't work, I can't pull any of the old branches. So to stop wasting any more time on this I will just create a new branch to work on, rather than update the current techstack branch.

kimc0de commented on content/ss23/master/m4-shop-gpt/challenges.md:
this file is already on main like this. why does github considered it updated?

huongvu2312 replied:
I did update the weight (order) of each tab.

kimc0de commented:
The title is future and features but it seems like you have only update content for features. Future is already done in main :D

huongvu2312 commented:
> The title is future and features but it seems like you have only update content for features. Future is already done in main :D

Yeah, I just check my change again. It seems on Friday I updated future change directly in main and updated features on techstack branch? What a mess, I'm so sorry.
